src/components/widgets/index.js

```diff
@@ -0,0 +1,2 @@
+import Button from './Button';
```

Comment:
I hadn't added a widget index since widgets are meant to be internal. That said I am using some in the batch app so apparently they are not that internal. I suppose we may as expose more widgets!

Reply:
we could put Dropdown in here eventually as well (and other useful components folks may wish to use in their apps)